content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md

```diff
@@ -96,6 +95,8 @@ You can define one or multiple `topologySpreadConstraint` to instruct the kube-s
   - `ScheduleAnyway` tells the scheduler to still schedule it while prioritizing nodes that minimize the skew.
 - **labelSelector** is used to find matching Pods. Pods that match this label selector are counted to determine the number of Pods in their corresponding topology domain. See [Label Selectors](/docs/concepts/overview/working-with-objects/labels/#label-selectors) for more details.
 
+When a Pod defines more than one `topologySpreadConstraint`, those constraints are ANDed: The kube-scheduler looks for a topology domain for the incoming Pod that satisfies all the constraints.
```

Comment:
the scheduler looks for nodes, not for topology domains. Nodes have topology domains.

Reply:
I think this sentence is ok. Although scheduler looks for nodes from implementation details perspective, logically the scheduler looks for a topology domain which might be node/zone/region. There are other similar statements in the doc like 'the scheduler placed this incoming Pod into "zoneA"'. We don't have to say 'the scheduler placed this incoming Pod into a node in "zoneA"'.

Reply:
But constraints are for different topology domains. So saying that a "topology domain satisfies all constraints" doesn't make much sense.

The correct interpretation is the "topology domains of a Node satisfy all constraints"